srcpkgs/n/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'n'
+pkgname=n
+version=9.1.0
+revision=1
+build_style=gnu-makefile
+depends="curl"
+short_desc="Simple command line NodeJS version management"
+maintainer="anelki <akierig@fastmail.de>"
+license="MIT"
+homepage="https://github.com/tj/n"
+changelog="https://raw.githubusercontent.com/tj/n/master/CHANGELOG.md"
+distfiles="https://github.com/tj/n/archive/refs/tags/v${version}.tar.gz"
+checksum=48306496413c61d37eeaa0a7328a4520b1da0c42739e046f6f9242de0d0ae270
+
+do_install() {
+	vbin bin/n n
```

Comment:
```suggestion
	vbin bin/n
```

Reply:
The makefile doesn't seem to respect $DESTDIR which means do_install would put it to the masterdir and not destdir.